db/db.go

```diff
@@ -363,6 +365,10 @@ func (db *Database) SearchAttribute(query url.Values) (results []types.SearchRes
 		where = append(where, fmt.Sprintf("states.tf_version LIKE '%s'", fmt.Sprintf("%%%s%%", v)))
 	}
 
+	if v := query.Get("lineage_value"); string(v) != "" {
```

Comment:
I doubt users will be searching by lineage since it's not a user-facing value. They're more likely to search by path (and maybe provider).

Reply:
The States aren't linked to the source provider atm, besides the providers do not even have an identifier or anything that would allow them to be identified.

> I doubt users will be searching by lineage since it's not a user-facing value

So I remove it? Tbh it doesn't take much space on the search view

Reply:
I'd remove it. I think mentions of lineage in the interface are more likely to be confusing for users than useful.